### DISCUSSION TITLE
3-dimensional image clasisfication issue

### DISCUSSION THREAD

**Discussion by dadaandpengpeng** [open-mmlab/mmclassification#1833] [Q&A]
Can I use mmpretrain to classifiy 3 dimensional images (such as CT or MRI volume)?

**Comment by JamesHoi** [comment]
I have similar question. Does mmpretrain support custom dataset such as 1-dimensional array and 3-dimensional image.
according to https://github.com/open-mmlab/mmselfsup/issues/727 , it said that mmselfsup is not support.
